XRTK-Core/Packages/com.xrtk.core/Definitions/Platforms/AllPlatforms.cs

```diff
@@ -9,6 +9,9 @@ namespace XRTK.Definitions.Platforms
     public sealed class AllPlatforms : BasePlatform
     {
         /// <inheritdoc />
-        public override bool IsAvailable => true;
+        public override bool IsActive => true;
```

Comment:
Why have we gone back to "isActive" when we had a whole discussion about changing to "IsAvailable", it's also in contradiction with the "IsBuildTargetAvailable" option.  

Can we revert this... again :D

Reply:
Because it's not active.

Being active is determined by the Service Locator when it gets all the types and resolves the list of `ActiveServices`

Reply:
See https://github.com/XRTK/XRTK-Core/pull/521/files#diff-5ebaa573e8b178b52c80b8c2c7f5873fR580

Reply:
Oh wait I see your point now lol. I'll revert it haha

Reply:
Updated @SimonDarksideJ thanks for the good set of eyes